data/zilliqa/assets/zil1vdc79fg75sth96xnt6xmq5f55swcd5t6m5zhvn/meta.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "YODA",
   "symbol": "YODA",
+  "web": "http://jody.tilda.ws",
+  "whitepaper": "http://jody.tilda.ws/whitepaper",
+  "supply": 10000,
+  "decimals": 5,
+  "email": "anzellkennard35@gmail.com",
+  "publicTeam": true,
```

Comment:
remove

Reply:
all 4 lines? or just publicTeam? is this on the basis of if I get company profiles for linkedin and facebook, then im able to have publicTeam?

Reply:
Just publicTeam. Why don't you make a company profile on linkedin and add yourself to it?

Reply:
Yep doing that now.. edit on the linkedin site coming soon.

Reply:
Hi Balthazar, unfortunately the requirements for a LinkedIn company page is far too great for my small project, it requires an email address with a company domain name, and several other things.  I will try and get them over time however.

I have updated my linkedin profile to show that Im the creator, I was wondering if this will suffice? It proves im the owner and that ive edited that page.

Reply:
Sounds good 😃